scheduler/src/cook/rate_limit.clj

```diff
@@ -14,8 +14,11 @@
 ;; limitations under the License.
 ;;
 (ns cook.rate-limit
-  (:require [cook.config :refer [config]]
+  (:require [clojure.tools.logging :as log]
+            [cook.config :refer [config]]
             [cook.rate-limit.generic :as rtg]
+            [cook.compute-cluster :as cc]
```

Comment:
Lexicographic

Reply:
Fixed

Reply:
Did it somehow not refresh correctly in your view?

https://github.com/twosigma/Cook/pull/1679/commits/af26db399497fea1bd9c077ede0c39257cbc155d has it changed, and its also correct in head.

Reply:
Shouldn't it be:

```
            [cook.compute-cluster :as cc]
            [cook.config :refer [config]]
```

Reply:
Odd. Fixed regardless; we're not using the compute cluster import, so it can be removed.